Answers.md

```diff
@@ -0,0 +1,18 @@
+
+1) my definition of a fluid layout would be a page that adjust "fluidly" based on the browsers with. It essentially is able to react in real time to the minimization or maximization of the viewport width. 
+
+  an adaptive design maintains a fixed layout until it hits a targeted breakpoint, in which, it will adjust to the specifications of the media query. 
+
```

Comment:
Yup spot on with this! I see you know that responsive with be more smooth as the width changes.
Adaptive with be more snappy.
✅